web-ui/src/main/java/org/archcnl/ui/input/ListEntry.java

```diff
@@ -0,0 +1,57 @@
+package org.archcnl.ui.input;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.archcnl.domain.input.datatypes.mappings.Concept;
+import org.archcnl.domain.input.datatypes.mappings.Relation;
+
+public class ListEntry<T> {
```

Comment:
I would choose a less generic name.

Reply:
Changed to TreeGridListEntry. Can't think of a more specific name as it is just a simple wrapper for its entries.

Reply:
Maybe something along the lines of `MappingListEntry`? And then the class could be made abstract and inherited by `ConceptListEntry` and `RelationListEntry` which implement the Concept/Relation specific behavior.